tcms/testplans/static/testplans/js/get.js

```diff
@@ -103,3 +117,100 @@ function getTestCaseExpandArea(row, testCase) {
         }
     });
 }
+
+function attachEvents(testPlanId, permissions) {
+    if (permissions['perm-change-testcase']) {
+        // update default tester
+        $('.js-test-case-menu-tester').click(function(ev) {
+            // todo this prompt need to be translated
+            var email_or_username = window.prompt('Please type new email or username');
```

Comment:
There's no easy way to translate strings in JavaScript. A quick solution I can think of is putting this somewhere in the HTML template. Can actually be a data attribute somewhere at the top of the page if you want.